.github/ISSUE_TEMPLATE/question.yml

```diff
@@ -18,19 +18,35 @@ body:
     attributes:
       label: "Issue I am facing"
       description: "Please describe the issue here in as much detail as possible"
+    validations:
+      required: true
       
   - type: textarea
     id: traceback
     attributes:
       label: "Traceback to the issue"
-      description: "This will be automatically formatted into code (shell), so no need for backticks."
-      placeholder: "put it here"
-      render: shell
+      description: "If you are facing a specific error message, please paste the traceback here. This will be automatically formatted into code (shell), so no need for backticks."
+      placeholder: |
+        Traceback (most recent call last):
+          File "/home/bot.py", line 1, in main
+            foo = bar()
+          ...
+        telegram.error.BadRequest: Traceback not found
+      render: python
         
   - type: textarea
     id: related-code
     attributes:
       label: "Related part of your code"
       description: "This will be automatically formatted into code (python), so no need for backticks."
       placeholder: "put it here"
-      render: python
\ No newline at end of file
+      render: python
+
+  - type: textarea
```

Comment:
Sorry, I should been more precise in https://github.com/python-telegram-bot/python-telegram-bot/pull/2669#discussion_r711642914 : Could you add the "config" section and the operating system (required) so this part is the same as in the bug-report temlate?